test/functional/interface_rest.py

```diff
@@ -223,6 +223,56 @@ def run_test(self):
         self.nodes[0].generate(1) #generate block to not affect upcoming tests
         self.sync_all()
 
+
```

Comment:
Be aware of #12766, which completely refactors this test (and makes it much better IMO). That obviously conflicts heavily with this change.